src/main/java/reposense/system/LogsManager.java

```diff
@@ -88,30 +91,75 @@ private static void removeHandlers(Logger logger) {
      * Creates {@link FileHandler} if it is null.
      */
     private static void addFileHandler(Logger logger) {
-        Path path = logFolderLocation.resolve(LOG_FOLDER_NAME);
+        Path path = TEMP_LOG_DIR.resolve(LOG_FOLDER_NAME);
 
         try {
-            if (!Files.exists(path)) {
+            boolean isFileExist = Files.exists(path);
+            if (!isFileExist) {
```

Comment:
should this and line 107 be combined? since this actually creates the log temp folder

Reply:
The reason why I shifted the logging down to 107, is that if the temp log folder is created, but the file handler is not created( which usually seems to be created together), the log won't be output. 

However, we need to create the temp log folder before the file handler can be created, as otherwise, an error will occur as the lock file, `reposense.log.0.lck` cannot be created as the directory does not exist.